app/containers/App/index.js

```diff
@@ -62,6 +62,7 @@ const Main = styled.div`
   bottom:0;
   overflow: ${({ isPrint }) => isPrint ? 'auto' : 'hidden'};
```

Comment:
actually this could now be limited to overflow-x

Reply:
https://github.com/marine-defrag/marine-defrag-client/commit/de21a624e6222cc36e845c4fc5d744204fd4504d